LDAR-Sim_inputs.md

```diff
@@ -661,17 +650,29 @@ If you are developing for LDAR-Sim, please adhere to the following rules:
 
 **Notes of caution:** Indicating a spin\_up value only affects plotting and does not affect the underlying data outputs. If you want to compare emissions of two programs using output datasets and want to exclude the influence of the initial step change in emissions, a spin up must be added manually.
 
-## start\_year
+## start\_date
 
-**Data type:** Integer
+**Data type:** List of Integers [year, month, day]
+
+**Default input:** [2017,1,1]
+
+**Description:** The date in which the simulations began.
+
+**Notes on acquisition:** No data acquisition required.
 
-**Default input:** 2003
+**Notes of caution:** The NetCDF file must encompass the start date and end date.
 
-**Description:** The year at which the simulation will begin.
+## end\_date
+
+**Data type:** List of Integers [year, month, day]
+
+**Default input:** [2020,12,31]
+
+**Description:** The date in which the simulations ends.
 
 **Notes on acquisition:** No data acquisition required.
 
-**Notes of caution:** The NetCDF file must encompass the start year plus the number of timesteps. For example, if the start date is 2015 and 1460 timesteps are used, the NetCDF file must extend from at least 2015 to 2019.
+**Notes of caution:** The NetCDF file must encompass the start date and end date.
```

Comment:
I thought we had it configured so that it would loop through weather data? If the weather data must incorporate the end date then we can no longer do projection?